cpp/ql/src/Security/CWE/CWE-428/UnsafeCreateProcessCall.ql

```diff
@@ -100,14 +100,14 @@ predicate isQuotedOrNoSpaceApplicationNameOnCmd(string s) {
 
 from CreateProcessFunctionCall call, string msg1, string msg2
 where
-  exists(Expr source, Expr appName, NullAppNameCreateProcessFunctionConfiguration nullAppConfig |
+  exists(Expr appName, NullAppNameCreateProcessFunctionConfiguration nullAppConfig |
     appName = call.getArgument(call.getApplicationNameArgumentId()) and
-    nullAppConfig.hasFlow(DataFlow2::exprNode(source), DataFlow2::exprNode(appName)) and
+    nullAppConfig.hasFlowToExpr(appName) and
```

Comment:
I think there's a behaviour change here (probably an improvement).  The original required that the source was an `exprNode`, whereas the new version does not.

Reply:
Good point! Strictly speaking this should then have been part of a different PR, but if you're OK with it we can let this slide, since as you said it seems beneficial.

Reply:
I'm happy (with all three of these), I'd be surprised if it makes any difference in practice, and if it does I'd expect a positive change. :+1:

Reply:
FWIW, `hasFlowToExpr` performs better than `hasFlow` after https://github.com/github/codeql/pull/10744.